src/tools/rustdoc-js/tester.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
I don't really understand the point of this little script (even with your explanations). You can already do it using the `x.py` script (which could be better documented I guess).